lib/zstd.h

```diff
@@ -332,6 +332,10 @@ typedef enum {
                               * The higher the value of selected strategy, the more complex it is,
                               * resulting in stronger and slower compression.
                               * Special: value 0 means "use default strategy". */
+    ZSTD_c_literalCompressionMode=108,  /* Controls how the literals are compressed (default is auto).
+                                         * The value must be of type ZSTD_literalCompressionMode_e.
```

Comment:
The documentation states that `The value must be of type ZSTD_literalCompressionMode_e`.

This implies that `ZSTD_literalCompressionMode_e` must also be part of "stable" public API.